community/blog/2022-10 Arcus Templates v1.0 a major release with major features/README.md

```diff
@@ -0,0 +1,91 @@
+# Arcus Templates v1.0: a major release with major features
+The Arcus Templates library is less prone to breaking changes as it does not provide code but starter projects. Because of this, major releases could be unnecessarily postponed. We decided to release Arcus Templates v1.0 as the number of features and changes is far too great for a minor release. Come see it for yourself in this run-down post of the release's highlights.
```

Comment:
I think I'd use other wording here.
Maybe focus on the fact that Arcus Templates is mature enough for a v1 release ?

Something like:
```
Arcus Templates has been around for a while now and was previously published with version-number 0.x.x.  Since we have created multiple projects using these templates, we feel Arcus Templates is now mature enough to be released a as a production ready v1 release
```
or something ?

Reply:
Aha, ok, that does seem like a better approach. Mine was maybe a bit too technical. 👍 
Let's use your suggestion.